src/omniscapeTransformer.py

```
@@ -850,6 +910,32 @@ def cleanup_julia_process():
+        merge_tile_outputs(tile_mod_paths, merged_mod_path, manifest['full_extent'])
+        myOutput.modifiedResistance = pd.Series(merged_mod_path)
+        ps.environment.update_run_log("  Added modifiedResistance output (merged tiles)")
+    elif is_tiling:
+        ps.environment.update_run_log(
+            "  Deferring modifiedResistance merge until tile aggregation is complete"
+        )
+    else:
+        mod_path = os.path.join(dataPath, "omniscape_ResistanceModifier", f"mod{last_mod_idx}-resistance.tif")
+        if os.path.exists(mod_path):
+            myOutput.modifiedResistance = pd.Series(mod_path)
+            ps.environment.update_run_log(f"  Added modifiedResistance output")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)